dotnet/src/webdriver/Chromium/ChromiumDriverService.cs

```diff
@@ -154,6 +159,15 @@ protected override string CommandLineArguments
                 argsBuilder.Append(string.Format(CultureInfo.InvariantCulture, " -allowed-ips={0}", this.AllowedIPAddresses));
             }
 
+            if (this.LogLevel != ChromiumDriverLogLevel.Default)
+            {
+                if (Enum.IsDefined(typeof(ChromiumDriverLogLevel), this.LogLevel))
```

Comment:
Unnecessary verification, it is already done at compile time.

Reply:
https://github.com/SeleniumHQ/selenium/pull/16139